orchestra/project_api/api.py

```diff
@@ -29,6 +30,61 @@ def get_project_information(project_id):
     }
 
 
+def get_projects_information(project_ids):
+    """
+    output format:
+    { project_id: {
+        'project': serialized_project_data,
+        'tasks': project_tasks_list,
+        'steps': project_workflow_steps
+        },
+        ...
+    }
+    Example:
+    {
+        123: {
+            'project': {},
+            'tasks': [],
+            'steps': []
+        },
+        ...
+    }
+    """
+    projects = Project.objects.select_related(
+        'workflow_version__workflow').filter(id__in=project_ids)
+    tasks = get_projects_tasks_data([p.id for p in projects])
+    projects_dict = defaultdict(dict)
+    for project in projects:
+        project_id = project.id
+        workflow_version = project.workflow_version
+        workflow = workflow_version.workflow
+
+        projects_dict[project_id]['project'] = ProjectSerializer(
+            project).data
+        projects_dict[project_id]['steps'] = get_workflow_steps(
+            workflow.slug, workflow_version.slug)
+        projects_dict[project_id]['tasks'] = tasks.get(project_id, {})
+    return projects_dict
+
+
+def get_projects_tasks_data(project_ids):
```

Comment:
Let's have only one function `get_projects_tasks_data` that accepts a list of project id as an argument.